.github/workflows/tests.yml

```diff
@@ -0,0 +1,185 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - run: npm ci
+      - run: npm run bootstrap -- --ci
+      - run: npm run lint
+
+  puppeteer:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node}}
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+      - run: npm run build --prefix=packages/puppeteer
+      # HACK: Force a TTY to enable browser-driver-manager to manipulate stdout.
+      - shell: 'script -q -e -c "bash {0}"'
```

Comment:
Should we use https://github.com/browser-actions/setup-chrome instead of relying on browser-driver-manager to install chrome? That would bypass the need for this hack to force TTY. Then browser-driver-manager could just install chromedriver which shouldn't need to manipulate the stdout.

Reply:
Possibly, I can play around with that.

FWIW if we didn't have the `chromedriver` dev dep here, we wouldn't need to do anything (see https://github.com/dequelabs/jazzband/pull/486).

Reply:
Ya, the need to have the npm chromedriver match the installed version of chrome causes so much pain.

Reply:
For jazzband, we just assume the developer (or CI container) has chromedriver on `$PATH`. Can we do the same here?

Reply:
No, because webdriverio has a direct dependency on the npm chromedriver [in order to setup the test](https://github.com/dequelabs/axe-core-npm/blob/develop/packages/webdriverio/test/axe-webdriverio.spec.ts#L58)

Reply:
Wait, looks like if it's on path we don't need chromedriver, so maybe you're correct?

Reply:
CLI would at least [need it or pass in the path manually](https://github.com/dequelabs/axe-core-npm/blob/develop/packages/cli/src/lib/webdriver.ts#L14). Not sure if letting it be on `$PATH` there would be sufficient.

Reply:
Using the `setup-chrome` option still requires us to have the correct `chromedriver` version installed. Since we're not going to remove the `chromedriver` dep (until we move over to `@puppeteer/browsers`), my vote is we keep this as-is.